helm_deploy/values-dev.yaml

```diff
@@ -9,6 +9,9 @@ generic-service:
 
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
+    PRISON_API_ENDPOINT_URL: https://api-dev.prison.service.justice.gov.uk
+    JAVA_OPTS: "-Xmx512m"
```

Comment:
do you really need the extra heap space?

Reply:
Probably not... I copied it from whereabouts api. I'll remove it